client/src/document/ingredients/browser-compatibility-table/feature-row.tsx

```diff
@@ -29,7 +29,8 @@ interface CompatStatementExtended extends bcd.CompatStatement {
 
 function getSupportClassName(
```

Comment:
Thank you! One last thing: Could we have a comment before this function explaining that we want to always show the partial symbol in the timeline (aka support history), whereas we want to show a "no support" symbol in the support cell if the current support is about a removed partial support.

A solution involving CSS changes would be to just return a new CSS class "removed_partial" and show a different symbol depending on the context. Actually, this might be the cleanest solution.

Reply:
Added CSS and a new class. Doesn't remove the need for the timeline boolean for CellIcons though, since that changes the title text and hidden text.